Multisig/Logic/Models/TokenRegistry/HardcodedTokenStore.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  HardcodedTokenStore.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 13.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct HardcodedTokenStore {
+
+    private var store = InMemoryTokenStore()
+
+    init() {
+        store.add(
+            Token(type: .erc20,
+                  address: AddressRegistry.ether,
+                  logo: BackendTokenStore.logo(AddressRegistry.ether),
```

Comment:
I would be for using our asset for ether rather than making a network call here

Reply:
but this needs to be a URL, how do I plug in image here?